packages/manager/modules/hub/src/variables.scss

```diff
@@ -43,6 +44,8 @@ $hub-border-radius-default: 6pt;
 
   &:hover {
     color: $hub-button-fg-color;
+    background-color: $p-700;
+    box-shadow: 0 4px 6px 0 rgba(0, 14, 156, 0.2);
```

Comment:
I think there is a variable for this but it can be done in a second time

Reply:
looks like p-700 with 20% opacity

Reply:
$base-box-shadow-secondary